aptos-move/e2e-move-tests/src/aggregator_v2.rs

```diff
@@ -57,6 +58,7 @@ pub fn initialize(
 
 pub struct AggV2TestHarness {
     pub harness: MoveHarness,
+    pub comparison_harnesses: Vec<MoveHarness>,
```

Comment:
I've used `MoveHarness` before by copy pasting existing code but never got a good understanding of what harness means. What does a harness mean exactly?

Reply:
What is comparison_harness? Doesn't seem to be used anywhere else in this PR.

Reply:
It represents a environment in which you can execute move code. It includes storage, block executor, etc.

So multiple harnesses here - mean multiple separate storage instances , on which we will execute same set of transactions, and compare that they are identical. For example we could initialize one harness with aggregators feature flag enabled, and another harness with feature flag disabled